### ISSUE TITLE
Users unable to log in if their email address is mismatched to the case they're entering

### ISSUE THREAD

**Issue by chcholman** [UM-USElab/gradecraft-development#3643]
We are continuing to occasionally hear about this bug - will you check that both the LTI method and the sorcery log in method downcase email/username before comparison?

**Comment by dovek** [comment]
I've looked into this via RSpec. On the user_sessions_controller for the method lti_create, I modified the code (for testing purposes only) to downcase the email from the auth_hash when attempting to auto_login. Using RSpec to test, I was able to log a user in who didn't have the same matching case on the auth_hash as the user had in the database:

```
=> #<User:0x007ff7036690e8
 id: 1,
 username: "bridie_pagac",
 email: "Bob.Ross@ross.org",
 crypted_password: "$2a$10$eh8h67c8ftcYVvSadJV2VeBDwU1gA12m6clVbqQn8oT6fbVxT.PAy",
```

vs.

```
=> #<User:0x007ff708420070
 id: 1,
 username: "bridie_pagac",
 email: "bob.ross@ross.org",
 crypted_password: "$2a$10$eh8h67c8ftcYVvSadJV2VeBDwU1gA12m6clVbqQn8oT6fbVxT.PAy",
```

All tests passed and this user was able to log in.

To reiterate, I've only tested this via RSpec and not against an actual Canvas implementation. At this point, I don't believe this to be an active bug. However, if it is brought up again, I feel the next steps should be testing through a practice canvas course.
